### ISSUE TITLE
[com.zergatul.cheatutils.controllers.WorldDownloadController] Large memory consumption for one class. 540MB of consumption with a total limit of 1024MB

### ISSUE THREAD

**Issue by freedom1b2830** [Zergatul/cheatutils#39]
Reproducing the bug:
1)Install ``-Xmx1024M`` (The limit is set for example)
2) Fly/Walk around the world
(In less than 5 minutes, braking will begin during GC cleaning)

``minecraft 1.20.1``
``cheatutils 2.7.1 fabric version (I haven't checked it on Forge)``

```
openjdk version "20.0.2" 2023-07-18
OpenJDK Runtime Environment (build 20.0.2+9)
OpenJDK 64-Bit Server VM (build 20.0.2+9, mixed mode, sharing)
```
540MB:
![gc1](https://github.com/Zergatul/cheatutils/assets/102818470/5f0ca102-2360-47e9-90b8-0596dedaca32)
![gc2](https://github.com/Zergatul/cheatutils/assets/102818470/defeb837-9ade-4f26-bc20-a5220f879663)

**Comment by Zergatul** [comment]
Thanks for the detailed report, I will look into it

**Comment by Zergatul** [comment]
This is fixed in 2.7.2

**Comment by freedom1b2830** [comment]
I confirm the leak has been fixed
